tests/scripts/validate_modified_files.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env -S bash
-set -ex
```

Comment:
why removing this?

Reply:
@subhamkrai  wrote:
> why removing this?

1. I prefer having explicit error exits accompanied by error messages  in scripts instead of implicit exits without helpful messages, due to `set -e`.
2. I find the tracing of `set -x` very confusing: it obfuscates the important messages the script prints and makes debungging very hard, IMHO.


short summay: `set -ex` makes script output extremely hard to understand for me and removing it is almost always the first thing I do to a script.  So it is mostly personal preference.